topcoffea/modules/corrections.py

```diff
@@ -21,24 +44,22 @@ def get_method1a_wgt_singlewp(eff,sf,passes_tag):
     wgt = p_data/p_mc
     return wgt
 
-# Evaluate btag sf from central correctionlib json
-def btag_sf_eval(jet_collection,wp,year,method,syst):
+def get_method1a_wgt_doublewp(effA, effB, sfA, sfB, cutA, cutB, cutC):
+    effA_data = effA * sfA
+    effB_data = effB * sfB
+
+    pMC = ak.prod(effA[cutA], axis=-1) * ak.prod(effB[cutB] - effA[cutB], axis=-1) * ak.prod(1 - effB[cutC], axis=-1)
+    pMC = ak.where(pMC==0,1,pMC) # removeing zeroes from denominator...
+    pData = ak.prod(effA_data[cutA], axis=-1) * ak.prod(effB_data[cutB] - effA_data[cutB], axis=-1) * ak.prod(1 - effB_data[cutC], axis=-1)
 
+    return pData, pMC
+
+# Evaluate btag sf from central correctionlib json
+def btag_sf_eval(jet_collection,wp,year,method,syst,fromPOG=True):
```

Comment:
I don't think `fromPOG` is used anywhere. Can the option be removed?

Reply:
@anpicci, wondering if you agree this option can be removed? If so, I think once it's removed we would be able to go ahead and merge.

Reply:
I removed the unused option. If there are any objections, we can always add it back in later.